index.html

```diff
@@ -6,216 +6,266 @@
+                of my work and  experience! If you like what you see and have a project
+                you need coded, don't hestiate to contact me.
             </p>
 
             <ul class="social-media">
                 <li>
-                    <a href=""><img src="./img/twitter.svg" alt="Twitter icon"></a>
+                    <a href=""><img src="./img/twitter.svg" alt="Twitter icon" /></a>
                 </li>
                 <li>
-                    <a href=""><img src="./img/Linkedin icon.svg" alt="Linkedin icon"></a>
+                    <a href=""><img src="./img/Linkedin icon.svg" alt="Linkedin icon" /></a>
                 </li>
                 <li>
-                    <a href=""><img src="./img/medium.svg" alt="Medium icon"></a>
+                    <a href=""><img src="./img/medium.svg" alt="Medium icon" /></a>
                 </li>
                 <li>
-                    <a href=""><img src="./img/github.svg" alt="Github icon"></a>
+                    <a href=""><img src="./img/github.svg" alt="Github icon" /></a>
                 </li>
                 <li>
-                    <a href=""><img src="./img/angelist.svg" alt="Angelist icon"></a>
+                    <a href=""><img src="./img/angelist.svg" alt="Angelist icon" /></a>
                 </li>
             </ul>
         </div>
     </section>
 
     <div class="disabled">
         <button type="button">
-            <img src="./img/chevron-down.svg" alt="disabled button">
+            <img src="./img/chevron-down.svg" alt="disabled button" />
         </button>
     </div>
 
-
     <section>
-        <div class="headline-title" style="padding: 0 30px;">
-            My Recent Works
+        <div class="headline-title" style="padding: 0 30px">
+            <h2>My Recent Works</h2>
         </div>
         <div class="divider-container">
             <div class="divider"></div>
         </div>
-        <div class="card-container" style="margin: 70px 0 20px 0;">
+        <div class="card-container" style="margin: 70px 0 20px 0">
             <div class="project-card">
                 <div class="descriptions">
-                    <div class="card-title"> Multi-Post Stories Gain+Glory</div>
-                    <div class="grid-container">
-                        <div class="item">Ruby on rails</div>
-                        <div class="item">css</div>
-                        <div class="item">JavScript</div>
-                        <div class="item">html</div>
-                    </div>
+                    <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
+                    <ul class="grid-container">
+                        <li class="item">Ruby on rails</li>
+                        <li class="item">css</li>
+                        <li class="item">JavScript</li>
+                        <li class="item">html</li>
+                    </ul>
                     <button class="see-project-btn">See Project</button>
```

Comment:
- [ ] - [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:
![hover](https://user-images.githubusercontent.com/66992901/201480443-7e5d2fb4-9a65-4b4e-8448-799db8f7c168.PNG)